tensorboard/webapp/metrics/store/metrics_types.ts

```diff
@@ -125,6 +125,12 @@ export type CardMetadataMap = Record<
   CardMetadata
 >;
 
+export type CardSettings = {
+  tableExpanded?: boolean;
```

Comment:
Does it make sense to make these none optional and then the action receives a Partial<CardSettings> object?

Reply:
Yeah, that's a great idea!